### PR TITLE
Hotfix/toggle message banner

### PR DESCRIPTION
**Pull request by nrwinner**
Screen Shot 2019-09-19 at 2 53 03 PM

### CODE REVIEW

File: src/app/components/navbar/components/message/message.component.scss [hunk 1]
```diff
@@ -1,44 +1,40 @@
...
+      margin-right: 10px;
     }
+  }
 
-    // styles
-    &.bad {
-        background: $error-red;
-        color: white;
-    }
+  // styles
```
Comment (nvisal1) [inline src/app/components/navbar/components/message/message.component.scss]: Please remove

File: src/app/core/messages.service.ts [hunk 2]
```diff
@@ -5,49 +5,51 @@ import { throwError } from 'rxjs';
...
+    } else {
+      return this.http.get(MISC_ROUTES.CHECK_STATUS, { withCredentials: true })
+        .pipe(
+          retry(3),
+          catchError(this.handleError)
+        )
+        .toPromise()
+        .then((val: Message) => {
+
+          // TODO remove this
```
Comment (nvisal1) [inline src/app/core/messages.service.ts]: Please make this more clear

File: src/app/components/navbar/components/message/message.component.scss (hunk 1 above)
Comment (nrwinner) [reply src/app/components/navbar/components/message/message.component.scss]: No

**Pull request merged**